### DISCUSSION TITLE
Disable - Select - Text Input Border

### DISCUSSION THREAD

**Discussion by Uhasith** [tusen-ai/naive-ui#5327] [Q&A]
How can I disable this blue color border in every select field when I search for something?

<img width="635" alt="Screenshot 2023-10-18 at 11 45 34" src="https://github.com/tusen-ai/naive-ui/assets/78094049/92bdcd46-1640-4c0b-95b5-1cc3e832a50d">

**Comment by Uhasith** [comment]
Can someone help?

**Comment by Volankey** [comment]
Unable to reproduce, please give a example.

**Comment by moseskamau338** [comment]
Am still facing this issue

**Reply by moseskamau338** [reply]
This solved it for me (using Tailwind CSS):
Am using Vue, and I just popped this into `styles` in the header

```css
.table-cell{
  @apply  whitespace-nowrap px-3 py-4 text-sm text-gray-500;
}
.n-base-selection-input:focus, .n-base-selection-input-tag__input:focus {
  @apply !ring-0;
}
```
